log-viewer/modules/TreeView.ts

```diff
@@ -492,6 +502,65 @@ function hideByDuration(elements: Array<HTMLElement>) {
   }
 }
 
+// Find the parent node of "elm" (or return null if we reach the root of the tree)
+function getParentNode(elm: Element | null) {
+  if (elm === null || elm.classList.contains('root')) {
```

Comment:
```suggestion
  if (elm?.classList.contains('root')) {
```

Reply:
Typescript does not believe that the suggested code will stop the next line from getting null's (even though it should).

Reply:
Actually typescript is correct...